get-coverage-percent/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "get-coverage",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "ncc build index.js --license licenses.txt",
```

Comment:
can this be renamed to third party notices like we have in our other repos?

Reply:
updated! it's still in the dist folder, I can move it one level up if that feels better